### DISCUSSION TITLE
Welcome to AI Chat Discussions!

### DISCUSSION THREAD

**Discussion by lilac** [rabrain/ai-chat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gustavofalcao1** [comment]
I was thinking about contributing to the "AI Chat Menu Bar", and only then did I realize that the [project shown](https://github.com/rabrain/ai-chat) is only for the website, not the desktop application.

I imagine that it is built in JS/TY compiled with ElectronJS, but I could be wrong, and since I am using it and really like it, I will try to help with suggestions, it is the most I can do without seeing the code.

I may seem a little vague in my suggestions, sorry for my non-native English, but my ideas and suggestions are accurate, as a developer and user.

> - [ ] 1. Small detail in the menu bar icon, it only has a black version, I don't know if it is an SVG, but it can be easily worked around, using Electron's 'nativeTheme' to identify the system theme and change the icon color or image, in the best case scenario choosing a color that adapts to light and dark mode. Sometimes it can be easy to lose the icon in a dark bar, this may seem simple, but it saves time.
> - [ ] 2. Although it would be an excellent idea to have a browser-based compilation of ElectronJS, so that you could have a fixed and separate window, it would still be very rewarding to be able to fix the floating window of the bar, even if to justify this, it would be a transparent window, highlighting only the chat bubbles.
> - [ ] 3. The shortcut keys could be customized, but to make it simpler, just make the "Toggle Chat Window" shortcut more comfortable to be used only with the left hand, leaving something like SHIFT+ALT+A/S/D or also CRTL+ALT+A/S/D.
> 
> - As a native Portuguese speaker, I am available to translate everything into that language, I like working on JS/TY projects and I need more experience in projects that are not created by me (working in a team), so I only have to gain from all this.

To conclude my first message in this community, I would like to congratulate the entire team involved for their excellent work. I am someone who is very critical of the tools I use, and this tool won me over right away, in such a way that for the first time, I am writing something to contribute to a project, even though I have been a developer for a long time. You are on the right track with this tool to make it, who knows, the intelligent co-pilot of the Linux world.

If I said anything wrong about the availability and the way this project is managed, I apologize, but still accept my suggestions if they make sense, I will be very grateful if you continue using and contributing to this project.

**Reply by lilac** [reply]
Thanks for your suggestions. I am glad you enjoy using the app. Item 1 & 3 are already in our roadmap, and for item 2 I don't fully understand the proposed changes.

Currently we are working on plans to sustain this product (AI Chat) in long term. Right now it's free, so we have to rely on other (paid) products to survive the organization. In future, we may consider open source as an option, so that others could contribute.

**Reply by vitor-dantas** [reply]
Hi @lilac! Congratulations for the app.

Are there estimates as to when items 1 and 3 from the top comment will be implemented? It would be very nice. Plus, the option to fix the bar above other windows even when it loses focus. 

Thanks

**Comment by ghost** [comment]
Thank you so much for this app, it's amazing! Could you please tell me how I can zoom in on the page? Ctrl++ doesn't seem to work.

**Comment by PaulG-Codersquad20** [comment]
Hello, App Creator,

I really appreciate you creating this app, I enjoy this as a primary A.I. control interface. I do have (1) question, is there a way you can make the interface available only in the toolbar menu and not an open app icon in the dock? I would enjoy the app being open in the control panel alongside internet control, audio, etc, at all times, while not docked alongside my open application icons. I hope this makes sense.

**Comment by clutz2000** [comment]
Hello. Just found this app today. Seems stable so far

Couple things though:
- I would like to see the app stay open once it loses focus, maybe an option to toggle this so that others that like it going away works too
- On ChatGPT, it seems that pressing Enter after typing doesn't send/execute the text. It works online when pressing Enter

Works great so far.. great job!!!

**Comment by dtseto** [comment]
Is this built using electron? I could not tell looking in the repositories. I do like your app since it has multiple different ai options. One feature I would like is a setting where if I click back to a window on my desktop it closes the ai chat. Sometimes I like to look at several windows and think. I dislike bulky electron on my older mac so I built a native mac desktop chatgpt client using the api directly.

**Comment by Argentum-Rex** [comment]
Hello, 

I've been testing it out for a while and I love to have so many different AI chatbots available in one package. So far I only have 2 gripes, which I'm sure they will be ironed out in the future.

• Being able to have it open when losing focus. A toggle, perhaps, as another commenter suggested.
• A white tray icon (it's quite hard to make if your taskbar is opaque).

![2024-12-16 18_04_47-](https://github.com/user-attachments/assets/4d98807a-effc-443f-bd0e-701a396c3e90)
